dashboard/app/helpers/levels_helper.rb

```diff
@@ -126,6 +125,12 @@ def app_options
       callouts: callouts
     )
 
+    # External project levels are any levels of type 'external' which use
+    # the projects code to save and load the user's progress on that level.
+    view_options(is_external_project_level: true) if @level.pixelation?
```

Comment:
We already have similar logic for Big Game (load data from the channel, don't display the full project header).  Can we re-use that here?  I would like to collapse new project types into existing ones, rather than creating new behaviors that we need to test & maintain.